executor/batch_point_get.go

```diff
@@ -313,6 +317,14 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
+		if hasKeys {
```

Comment:
Non RC isolation level needs to update the delta map too in code block L270-L283.

Reply:
Updated PTAL